MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -126,7 +132,7 @@ final class MovieQuizViewController: UIViewController {
 
     private func showAnswerResult(isCorrect: Bool) {
         if isCorrect {
-                correctAnswers += 1
+            correctAnswers += 1
             }
```

Comment:
Выделить код и зажать control+i - весь файл поправишь
```suggestion
        }
```